__tests__/app.test.js

```diff
@@ -84,8 +84,61 @@ describe("GET /api/articles", () => {
                     comment_count: expect.any(String),
                 })
                 expect(article).not.toHaveProperty('body')
-                expect(articlesArray).toBeSorted()
+                expect(articlesArray).toBeSortedBy("created_at", { descending: true})
             })
         })
     })
 })
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("status 200: responds with array of all comments with the correct properties", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+            const commentsArray = res.body.comments
+            expect(commentsArray).toHaveLength(11)
+            commentsArray.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
We should be able to expect 1 on this object here since you're requesting that article id